atlp-rwanda/ecommerce-app-champions-fe
# #83 Feature- implement notification panel

#### What does this PR do?
The notifications panel displays a user's notifications and prompts them to take action when they interact with notification icon
#### Description of Task to be completed

- The notifications pane should open up when a user clicks on the icon. It should contain a list of notifications
#### How should this be manually tested?
clone the repo using git clone repo's URL
change branch git ft- implement notification panel
create .env file and paste variables from .env.example
run npm run dev
login as a vendor
You should be redirected to the dashboard
click on the notification icon and you should be able to see the notification
